src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Repositories/CustomerDemoDbContext.cs

```diff
@@ -0,0 +1,27 @@
+using EntityFrameworkCore.Triggers;
+using Microsoft.EntityFrameworkCore;
+using VirtoCommerce.CustomerModule.Data.Repositories;
+using VirtoCommerce.DemoSolutionFeaturesModule.Data.Models;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Data.Repositories
+{
+    public class CustomerDemoDbContext : CustomerDbContext
+    {
+        public CustomerDemoDbContext(DbContextOptions<CustomerDemoDbContext> options)
+          : base(options)
+        {
+        }
+
+        protected CustomerDemoDbContext(DbContextOptions options)
```

Comment:
Are we really need overloaded protected constructor?

Reply:
No. It was already here.